docs/io/fits/index.rst

```diff
@@ -47,6 +47,10 @@ Once the `astropy.io.fits` package is loaded using the standard convention
 
     >>> hdul = fits.open(fits_image_filename)
 
+Note : The ``astropy.io.fits.util.get_testdata_filepath`` function is only used by tests to get their data.
+Please only use ``astropy.io.fits.open`` along with the path to the appropriate FITS data file
+which has to be taken from source.
+
```

Comment:
A few comments:

- There are also the comment lines in the above code block, I think we should not keep both as it can be confusing. 

- We could use a sphinx note (` .. note::`, see example above).

- "only used by tests to get their data" is misleading, as obviously it is not only used by tests. Maybe "is used to access test data shipped with Astropy" ?

Reply:
Also I don't think that this need to be repeated below.